translations/en-us.yaml

```diff
@@ -338,7 +338,7 @@ general:
   timedReleaseNoSchedule: "Available immediately when published"
   timedReleaseOnlyEndDate: "Available until {endDate}"
   timedReleaseOnlyStartDate: "Available: {startDate}"
-  timedReleaseStartAndEndDate: "Available: {startDate} and available until {endDate}"
```

Comment:
Does this need to change in the French and Spanish translations as well?

Reply:
I had tested this in Spanish before starting on this work and found it was fine - same with French. This issue was only in English.